modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```
@@ -881,28 +881,24 @@ Resources:
                   - UseWamStandalone
                   - !Sub |
                       # Download watchmaker standalone pkg
-                      [Net.ServicePointManager]::SecurityProtocol = "Ssl3, Tls, Tls11, Tls12"
+                      [Net.ServicePointManager]::SecurityProtocol = "Tls12"
```

Did you try including Tls13 also, by any chance? I think it ought to work, but possibly might not on ws2012....
```suggestion
                      [Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls13"
```

---

I did.  `Tls13` isn't an available option and caused an error in all the Windows tests.

---

That is super weird. It definitely works fine on my Windows 10 box 🤷‍♂️ this is in my powershell profile:

```
[Net.ServicePointManager]::SecurityProtocol = "Tls11, Tls12, Tls13"
```

---

Yeah.  Super weird.  I was surprised even Windows Server 2019 failed.  `Tls13` isn't an option.  Here's a quick test on the RDSH instance:

```
PS C:\Windows\system32> [Net.ServicePointManager]::SecurityProtocol = "Tls11, Tls12, Tls13"
Exception setting "SecurityProtocol": "Cannot convert value "Tls11, Tls12, Tls13" to type "System.Net.SecurityProtocolType". Error: "Unable to match the identifier name Tls11, Tls12, Tls13 
to a valid enumerator name. Specify one of the following enumerator names and try again:
SystemDefault, Ssl3, Tls, Tls11, Tls12""
At line:1 char:1
+ [Net.ServicePointManager]::SecurityProtocol = "Tls11, Tls12, Tls13"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], SetValueInvocationException
    + FullyQualifiedErrorId : ExceptionWhenSetting
 
PS C:\Windows\system32> [Net.ServicePointManager]::SecurityProtocol = "Tls11, Tls12"

PS C:\Windows\system32>
```

---

Dang. Really appreciate your thoroughness! Was just hoping to future-proof this a little bit. :sigh: